app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1398,27 +1443,53 @@ class BrowserTabViewModel(
         }
     }
 
-    fun onAddFavoriteMenuClicked() {
-        val url = url ?: return
-        val title = title ?: ""
-
-        val buttonHighlighted = currentBrowserViewState().addFavorite.isHighlighted()
-        pixel.fire(
-            AppPixelName.MENU_ACTION_ADD_FAVORITE_PRESSED.pixelName,
-            mapOf(FAVORITE_MENU_ITEM_STATE to buttonHighlighted.toString())
-        )
+    private suspend fun editSiteBookmark(url: String) {
```

Comment:
`url` is not used inside the method (I know we used it before).
 
a) We can replace the function parameter type on this method and receive a `SavedSite`. In that case, we could just use `onEditSavedSiteRequested(savedSite)` instead of this method.

b) We remove the function parameter and the code stays as it is.

I'm fine with both approaches.

Reply:
Good catch! Updated.